packages/checklist-block/src/types.ts

```diff
@@ -1,13 +1,13 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { ButtonSize, Color, ItemDragState, OrderableListItem } from '@frontify/fondue';
+import { ButtonSize, Color, LegacyItemDragState, OrderableListItem } from '@frontify/fondue';
```

Comment:
Probably you want to use `LegacyOrderableListItem` as well

Reply:
you are right, adjusted all of them